cli-feed-v3.json

```diff
@@ -17,7 +17,7 @@
       "hidden": true
     },
     "v2": {
-      "release": "2.11.1",
+      "release": "2.11.3",

```

Comment:
Did you mean to add this to the prerelease tag rather than here? 
@soninaren -- we are ready to move 2.11.2 to a real release now, right? Can we move the official release to 2.11.2 and the prerelease to 2.11.3? Or should we just skip straight to 2.11.3?